spec/index.bs

```diff
@@ -12840,6 +12842,19 @@ GPUQueue includes GPUObjectBase;
                     </div>
                 1. Issue: Do the actual copy.
             </div>
+            <div class=note heading>
+                When copying from a `"webgl"`/`"webgl2"` context canvas, the
```

Comment:
Huh, something seems to have gone wrong with handling of `` ` `` in our spec, as these are not rendering correctly. This is not the only place though, so we can look into this separately. Guessing this was a Bikeshed change.